### PR TITLE
ADEN-7103 fix vuap tests

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 1]
```diff
@@ -146,9 +146,7 @@ private void clickByJS(By selector) {
 
   private void clickElementInsideAd(By selector) {
     usingAdFrame(() -> {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: MAJOR Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 2]
```diff
@@ -144,11 +145,17 @@ private void clickByJS(By selector) {
...
+  private void clickOnBottomLeftCornerOfElement(WebElement element) {
+    int middlePointOfWebElement = element.getSize().height / 10;
+    int leftSideOfWebElement = element.getSize().width / 3;
+    Actions builder = new Actions(driver);
+    builder.moveToElement(element, middlePointOfWebElement,leftSideOfWebElement).click().build().perform();
+
+  }
+
   private void clickElementInsideAd(By selector) {
     usingAdFrame(() -> {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: MAJOR Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java [hunk 3]
```diff
@@ -144,11 +145,16 @@ private void clickByJS(By selector) {
...
 
+  private void clickOnBottomLeftCornerOfElement(WebElement element) {
+    int bottomPointOfWebElement = element.getSize().height - element.getSize().height / 10;
+    int leftPointOfWebElement = element.getSize().width / 10;
+    Actions builder = new Actions(driver);
+    builder.moveToElement(element, bottomPointOfWebElement,leftPointOfWebElement).click().build().perform();
+  }
+
   private void clickElementInsideAd(By selector) {
     usingAdFrame(() -> {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java]: MAJOR Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1602)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java [hunk 4]
```diff
@@ -9,6 +9,9 @@
 import org.openqa.selenium.WebElement;
 
 public class AdsFandomObject extends AdsBaseObject {
+  private static final By F2_MOBILE_NAVIGATION_BAR = By.cssSelector(".global-navigation-mobile");
+  private static final By F2_DESKTOP_NAVIGATION_BAR = By.cssSelector(".wds-global-navigation");
```
Comment (v3rth) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java]: I think as far as those consts are in `AdsFandomObject` we don't need to add F2 prefix to it

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java [hunk 5]
```diff
@@ -25,6 +28,16 @@ public void triggerOnScrollSlots() {
     jsActions.scrollToBottom();
   }
 
+  public void fixScrollPositionByNavbarOnF2(boolean isMobile) {
+    if(isMobile){
```
Comment (v3rth) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java]: Both cases behind `if` have common logic, you are using `if` statement only to determine which element use, so let's refactor it

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java [hunk 6]
```diff
@@ -46,6 +57,11 @@ public By getIframeSelector(String slotName) {
     return By.cssSelector(AdsContent.IFRAME_SLOT_SELECTORS.getOrDefault(slotName, getDefaultIframeSelector(slotName)));
   }
 
+  private void scroll_navigation_bar_height(By element){
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestAdsVuapFandom.java#L22](https://github.com/Wikia/selenium-tests/blob/784c677207dfcceb3a6278715135d002ad5ee3eb/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java#L22): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java (hunk 5 above)
Comment (bulaszenko) [reply src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsFandomObject.java]: Done fixed

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestAdsVuapFandom.java#L22](https://github.com/Wikia/selenium-tests/blob/7445a489a37ebd9d3cd0c8e29b28617cdc945c0f/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java#L22): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [TestAdsVuapFandom.java#L22](https://github.com/Wikia/selenium-tests/blob/dc6e14e378f95beabdd1ea0fc48b3d461e5fc123/src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandom.java#L22): Move this trailing comment on the previous empty line. [rule](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**